packages/shared/src/constants.tsx

```diff
@@ -7,12 +7,18 @@ import {
     OpenSeaColoredIcon,
 } from '@masknet/icons'
 
+export const MINDS_ID = 'minds.com'
+export const FACEBOOK_ID = 'facebook.com'
+export const TWITTER_ID = 'twitter.com'
+export const INSTAGRAM_ID = 'instagram.com'
+export const OPENSEA_ID = 'opensea.io'
+
```

Comment:
```
export DASHBOARD_ID = 'mask.dashboard'
export POPUP_ID = 'mask.popup'
```

Reply:
Fixed in https://github.com/DimensionDev/Maskbook/pull/5632/commits/88f16ca9f04c7bb114badf98103f05f3ee48bbb2